vertical-pod-autoscaler/e2e/v1/actuation.go

```diff
@@ -278,6 +287,51 @@ var _ = ActuationSuiteE2eDescribe("Actuation", func() {
 		ginkgo.By(fmt.Sprintf("Waiting for pods to be evicted, hoping it won't happen, sleep for %s", VpaEvictionTimeout.String()))
 		CheckNoPodsEvicted(f, MakePodSet(podList))
 	})
+
+	ginkgo.It("omits injected sidecars", func() {
+		const (
```

Comment:
Any reason why these const are separate from the anghost image and sidecarParam?

Reply:
All consts moved to the test case.